.github/workflows/e2e-tests.yml

```diff
@@ -42,6 +42,23 @@ jobs:
       - name: Add MSBuild to PATH
         if: inputs.os == 'windows-latest'
         uses: microsoft/setup-msbuild@v1
+      - name: Install Microsoft Visual C++ Redistributable
+        if: inputs.os == 'windows-latest'
+        run: |
+          choco install vcredist-all
+        shell: powershell
```

Comment:
```suggestion
```
Isn't this unnecessary? Default shell on Windows runners is `pwsh` anyway, which I assume should behave similarly; if it's not available, it would fall back to `powershell`. See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell.

Reply:
Good spot.